app/error.js

```
@@ -0,0 +1,40 @@
+'use client';
+
+import { useEffect } from 'react';
+import Link from 'next/link';
+
+export default function Error({ error, reset }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)